config/initializers/assets.rb

```diff
@@ -3,7 +3,7 @@
 # Be sure to restart your server when you modify this file.
 
 # Version of your assets, change this if you want to expire all your assets.
-Rails.application.config.assets.version = '1.1'
+Rails.application.config.assets.version = '1.0'
```

Comment:
"Downgrade" because bootstrap explicitly required this version? I assume `1.0` is the standard value.

Reply:
There is no dependency for the downgrade. Rather, it is shown each time an app upgrade is performed as a difference that one must skip.

Back in 2022, we increased the asset version to force a rebuild of all assets, caused by the introduction of Subresource Integrity (SRI). See 65c95a1f for details.

On our production deployment, we only keep the last five releases (and we've deployed more often since than) and also instruct browsers to cache assets for one year (which passed as well). Hence, I decided to align the version number again with the Rails defaults to make upgrades easier. It will, again, cause all assets to be rebuild. However, given the changes performed in #2497, that's actually not too bad.

Is that fine for you?

Reply:
I see, fine for me.